src/YesSql.Core/Store.cs

```diff
@@ -84,6 +87,11 @@ internal Store(IConfiguration configuration) : this()
 
         public async Task InitializeAsync()
         {
+            if (_state == StoreState.Initialized)
```

Comment:
Should just be a `private bool _isInitialized` for now. No need for a new enum.

Reply:
sure. Done